INSTALL.md

```diff
@@ -0,0 +1,81 @@
+# balena CLI Installation Instructions
+
+The easiest and recommended way of installing the CLI on all platforms (Linux, MacOS, Windows) is
+to use the [Standalone Installation](#standalone-installation) described below. Some specific CLI
+commands like `balena ssh` and `balena sync` have additional dependencies with separate
+installation steps: see section [Additional Dependencies](#additional-dependencies).
+
+> **Windows users:** we now have a [YouTube video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA)
+for installing and getting started with the balena CLI on Windows!
+
+## Standalone Installation
```

Comment:
I wonder if it would be better to have a section for each OS and then have links down to them like:

Standalone Installation:

- on windows (link to windows specific section)
- on macOS
- on Linux.

then those sections can be much more specific and users only have to read stuff that is specific to them.

Reply:
(Click the `View File` button on the header of `INSTALL.md` here on the GitHub PR review page, for a rendered view of the markdown.)